article-list.html

```diff
@@ -7,44 +7,94 @@
 
 	<body>
 		<header>
-			<p>This will be the nav</p>
+			<a href="index.html" class="logo">Skinclusive</a>
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
             <!-- This div is for the list of articles on the page -->
             <div class="article-list">
                 <h1>All Articles</h1>
+
+				<!-- This is the category subheading that will change depending on which section of page user is on -->
+				<p class="subheading">Read articles about common skin issues</p>

```

Comment:
In the final layout, I decided to also put subheadings that categorise the articles underneath, so the user knows what sort of articles are in each category (e.g. a subheading for "common skin issues" and below will have articles about common skin issues and how to deal with them).